runtime/lua/vim/treesitter.lua

```diff
@@ -612,4 +622,13 @@ function M.show_tree(opts)
   })
 end
 
+--- Returns the fold level for lnum in the current buffer. Can be set directly to 'foldexpr':
+--- <pre>lua
+--- vim.wo.foldexpr = 'v:lua.vim.treesitter.foldexpr()'
+--- </pre>
+---@param lnum integer|nil
```

Comment:
```suggestion
---@param lnum integer|nil Line number to calculate fold level for
```

missing `---@return` annotation?